File: Entities/AutomaticThought.cs
using System.Collections.Generic;

namespace MindYourMoodWeb.Entities
{
    public class AutomaticThought : Entity
    {
        //public int Id { get; set; }
        public bool HotThought { get; set; }
        public string Thought { get; set; }
        public ThoughtRecord ThoughtRecord { get; set; }
        public IEnumerable<EvidenceForHotThought> EvidenceForHotThought { get; set; }
        public IEnumerable<EvidenceAgainstHotThought> EvidenceAgainstHotThought { get; set; }
    }
}

File: ClientApp/src/app/_model/automaticThought.ts
import { EvidenceAgainstHotThought } from './evidenceAgainstHotThought';
import { EvidenceForHotThought } from './evidenceForHotThought';
import { ThoughtRecord } from './thoughtRecord';

export interface AutomaticThought {
  id: number;
  hotThought: boolean;
  thought: string;
  thoughtRecord: ThoughtRecord;
  evidenceForHotThought: EvidenceForHotThought[];
  evidenceAgainstHotThought: EvidenceAgainstHotThought[];
}
